src/drive/web/modules/filelist/FileOpener.jsx

```diff
@@ -80,6 +80,10 @@ const FileOpener = ({
             ev.srcEvent.stopImmediatePropagation()
             toggle(ev.srcEvent)
           } else {
+            // We need to filter here, is the event target the FileOpener or a link INSIDE the FileOpener?
+            // Assuming the FileOpener itself will never be inside an anchor, we can use the following guard
+            if (ev.srcEvent.target.closest('a')) return
```

Comment:
Should we put this logic in enableTouchEvents() method?

Reply:
hum yeah I guess so, we can put it in the first guard (L32)

Reply:
actually line 45 was trying to do that it seems, I guess it's not working anymore for some reason

Reply:
updated